proto/p4/v1/p4runtime.proto

```diff
@@ -412,10 +423,20 @@ message MulticastGroupEntry {
 // packet_length_bytes field is 0, no truncation on the clone(s) will be
 // performed.
 message CloneSessionEntry {
+  // Set session_id to CLONE_SESSION_ID_WILDCARD for wildcard reads
   uint32 session_id = 1;
   repeated Replica replicas = 2;
   uint32 class_of_service = 3;
   int32 packet_length_bytes = 4;
+
+  // We define a special reserved value for the session_id field. This value can
+  // be used to do a wildcard read on all configured clone sessions. This value
+  // cannot be used in Write RPCs.
+  enum CloneSessionIdReserved {
+    CLONE_SESSION_ID_FIRST = 0;
+    // 0xffffffff
+    CLONE_SESSION_ID_WILDCARD = -1;
```

Comment:
Maybe we should just spell this out as 0xffffffff, especially since negative numbers aren't actually valid IDs (they are uint32).

Reply:
Not possible in Protobuf IIRC, it will be rejected by protoc